server.js

```diff
@@ -3586,6 +3586,31 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// bitHound integration
+camp.route(/^\/bithound\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `github/rexxars/sse-channel`.
+  var format = match[2];
+  var url = 'https://www.bithound.io/' + userRepo + '/badges/score.svg';
+  var badgeData = getBadgeData('bithound', data);
```

Comment:
Ah, I misunderstood then. The figure isn't a percentage, I take it? We could put "quality".